src/components/modals/SettingsModal.tsx

```diff
@@ -12,15 +16,10 @@ import {
   ModalOverlay,
   Checkbox,
 } from "@chakra-ui/react";
+import mixpanel from "mixpanel-browser";
+import { ChangeEvent, memo } from "react";
 
-import { LabeledSelect, LabeledSlider } from "../utils/LabeledInputs";
-
-import { APIKeyInput } from "../utils/APIKeyInput";
-import { Settings, FluxNodeType } from "../../utils/types";
-import { getFluxNodeTypeDarkColor } from "../../utils/color";
-import { DEFAULT_SETTINGS, SUPPORTED_MODELS } from "../../utils/constants";
-
-export const SettingsModal = React.memo(function SettingsModal({
+export const SettingsModal = memo(function SettingsModal({
```

Comment:
when did we memo this hmmm wouldnt the fact that we're passing a function and object render that useless?

Reply:
hmmm i guess it works since the functions are setstate functions and thus memoized by react, and the objects are also products of useState so the same logic applies